league/flysystem-bundle/1.0/var/storage/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Wouldn't this directory be created automatically since it is in the "var" folder?

Reply:
Not sure what you mean, I don't think it will.